style.css

```
@@ -3655,11 +3680,13 @@ input:checked + .toggle:before {
 }
 
 .shortcutSettingsEntry input:hover {
-    animation: clipText 0s 150ms forwards;
+	animation: clipText 0s 150ms forwards;
 }
 
 @keyframes clipText {
```

Stylelint (17.10.0)

[error] Expected keyframe name "clipText" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)